memgpt/local_llm/koboldcpp/settings.py

```diff
@@ -0,0 +1,25 @@
+from ...constants import LLM_MAX_TOKENS
+
+# see https://lite.koboldai.net/koboldcpp_api#/v1/post_v1_generate
+SIMPLE = {
+    "stop_sequence": [
+        "\nUSER:",
+        "\nASSISTANT:",
+        "\nFUNCTION RETURN:",
+        "\nUSER",
+        "\nASSISTANT",
+        "\nFUNCTION RETURN",
+        "\nFUNCTION",
+        "\nFUNC",
+        "<|im_start|>",
+        "<|im_end|>",
+        "<|im_sep|>",
+        # '\n' +
+        # '</s>',
+        # '<|',
+        # '\n#',
+        # '\n\n\n',
+    ],
+    "max_context_length": LLM_MAX_TOKENS,
+    "max_length": 512,
```

Comment:
@Drake-AI is there a good reason to set this, vs leaving it at the default?

Reply:
I think default context is 2048. But it can be up to 32k. Not sure about default lenght but i think it is pretty low.

Reply:
Oh sorry it didn't tag correct - I was asking about `max_length` (which I presume is the max number of tokens to generate), not `max_context_length`

Reply:
80 tokens max_length by default

Reply:
Great OK so setting a higher value (still reasonable like 512) makes a lot of sense. Thank you @Drake-AI

Reply:
Thank you and your team for the great work.

Reply:
You're welcome! Thank you for the great PR!